.github/workflows/third_party.yml

```diff
@@ -60,19 +60,18 @@ jobs:
         uses: actions/checkout@v4
         with:
           path: typing-extensions-latest
-      - name: Setup pdm for pydantic tests
-        uses: pdm-project/setup-pdm@v4
-        with:
-          python-version: ${{ matrix.python-version }}
-          allow-python-prereleases: true
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        run: uv python install ${{ matrix.python-version }}
       - name: Add local version of typing_extensions as a dependency
-        run: pdm add ./typing-extensions-latest
+        run: uv add --editable ./typing-extensions-latest
```

Comment:
note: `--editable` isn't strictly required here. I included it as a sanity check. This causes the path to the installed `typing_extensions` to be printed in the later steps, which helps to confirm that the latest tip is being used (and not a version from PyPI)